vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/launch/accel_brake_map_calibrator.launch.xml

```diff
@@ -24,6 +24,8 @@
     <param name="use_sim_time" value="$(var use_sim_time)"/>
   </node>
 
+  <!-- calibration plot and map server node -->
+  <node pkg="accel_brake_map_calibrator" exec="plot_csv_server_node.py" name="plot_csv_server" output="screen"/>
```

Comment:
Change this node name more easy to understand that this node provide accel brake map calibrator result.

Reply:
Changed the name to "new_accel_brake_map_server"